tests/helpers.py

```diff
@@ -201,6 +206,40 @@ def create_pod(
 
     if sa_name and dc_deployment:
         pod_data['spec']['template']['spec']['serviceAccountName'] = sa_name
+
+    # disconnected cluster related configuration updates
+    if config.DEPLOYMENT.get('disconnected'):
+        if 'containers' in pod_data['spec']:
+            container = pod_data['spec']['containers'][0]
+        else:
+            container = pod_data['spec']['template']['spec']['containers'][0]
+        container['image'] = mirror_image(container['image'])
```

Comment:
This could be a separate function in ocs_ci module, so that this code don't have to be placed in fio_job_dict_fixture again. Other code could use it when necessary, and adding a unittest for this could be possible.

Reply:
Addressed by 63b8f65c97a6772b8281acda8e59e7f3cf9f2b60.